packages/block-library/src/details-summary/style.scss

```diff
@@ -0,0 +1,6 @@
+.wp-block-details-summary {
+	cursor: pointer;
+	&::marker {
+		margin-right: 0.5rem;
```

Comment:
@aaronrobertshaw I don't think the margin is applied. 
If I add line height, color etc here, it works.
But if I add for example 9rem instead of 0.5, there is no visible difference.
https://developer.mozilla.org/en-US/docs/Web/CSS/::marker

Reply:
Understood, I was [suggesting](https://github.com/WordPress/gutenberg/pull/45055#discussion_r1124221743) that perhaps the `::-webkit-details-marker` vendor prefix might provide a tool to make things uniform across browsers. An alternative also is to hide the marker and use `::before` or `::after`, leveraging `[open]` to change the custom marker based on state.

Either way its a small detail that can be iterated on if need be later and shouldn't be a blocker to this PR.